internal/controllers/topology/cluster/patches/variables/value.go

```diff
@@ -31,6 +31,24 @@ const (
 	rightArrayDelim = "]"
 )
 
+type variableNotFoundErr struct {
```

Comment:
This is an early attempt at making this error discoverable from an extension. To work properly it would need to be put in some importable package (errors maybe?) For now I'm duplicating the string, but using errors.Cause() to pick up the result.

Reply:
The whole util func should be in a public package not only the error :) (and then the error belongs in that package)

Reply:
What util function are you referring to here?

Reply:
`GetVariableValue`